dev-tools/reconfigurator-exec-unsafe/src/main.rs

```diff
@@ -278,11 +278,11 @@ impl ReconfiguratorExec {
 
             loop {
                 let status = status_rx.borrow();
-                debug!(&log, "MGS update status"; "status" => ?status);
+                info!(&log, "MGS update status"; "status" => ?status);
```

Comment:
should these be changed back to debug?

Reply:
Good question but I don't think so.  The intent was to print these every few seconds after execution finishes until the updates are complete.  Otherwise you don't really know what's going on with the updates.